tests/test_github.py

```diff
@@ -15,6 +15,9 @@
 ARBITRARY_CREATED = (
     datetime.datetime.utcnow() - datetime.timedelta(hours=1)
 ).replace(tzinfo=pytz.UTC, microsecond=0)
+ARBITRARY_CLOSED = (
+    datetime.datetime.utcnow() - datetime.timedelta(hours=2)
+).replace(tzinfo=pytz.UTC, microsecond=0)
```

Comment:
You are closing this issue 1 hour **before** you create it. It does not effect any tests because there aren't any yet, but I would invert these, so we don't bump our heads later.

Reply:
Ah, good catch! I did not pay enough attention. Thanks!